### DISCUSSION TITLE
Row DnD between tables

### DISCUSSION THREAD

**Discussion by geniusit** [KevinVandy/material-react-table#131] [Ideas]
It would be nice to allow row drag and drop between two tables.
Like https://ag-grid.com/javascript-data-grid/row-dragging-to-grid/

**Comment by KevinVandy** [comment]
There's an example in the MRT docs just for that 

https://www.material-react-table.com/docs/guides/row-ordering-dnd#drag-and-drop-rows-to-other-ui-or-tables
